src/pages/Jam.svelte

```diff
@@ -85,42 +83,40 @@
         updatePitch();
     }
 
-    let bufLen = 2048;
-    let buf = new Float32Array(bufLen);
-
     function updatePitch() {
-        // TODO: Maybe replace autoCorrelate with something like this:
-        const { thresholdMet } = freqAnalyze(analyser, sensitivity, true);
+        // TODO: Not working as expected
```

Comment:
What's happening and what's expected? I honestly forgot. Let's update this comment and/or add an issue describing exactly what we need to fix